app/components/syntax-highlighted-diff.ts

```diff
@@ -160,28 +177,53 @@ export default class SyntaxHighlightedDiffComponent extends Component<Signature>
     }
   }
 
-  highlightCode = task({ keepLatest: true }, async (): Promise<void> => {
-    if (this.isDarkMode === undefined) {
-      return; // Not ready to highlight yet
+  highlightCode = task({ [HIGHLIGHT_CODE_TASK_TIMEOUT_METHOD]: true }, async (): Promise<void> => {
+    // For "restartable" to have immediate effect for sequential calls
+    if (HIGHLIGHT_CODE_TASK_TIMEOUT_METHOD === 'restartable') {
```

Comment:
Ignoring this since we're most likely removing restartable

Reply:
Removed in favour of `keepLatest` method